packages/cli/src/commands/codegen.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable prefer-const */
+import { fixParameters } from "../lib/helpers/parameters";
+import { CodeGenerator } from "../lib/generators/CodeGenerator";
+
+import chalk from "chalk";
+import axios from "axios";
+import { GluegunToolbox } from "gluegun";
+
+const defaultTemplate = "web3api.gen";
+const defaultManifest = "web3api.yaml";
+
+const HELP = `
+${chalk.bold("w3 codegen")} ${chalk.bold("[<template-file>]")} [options]
+
+Template file:
+  Path to input template file (default: ${defaultTemplate})
```

Comment:
I think naming this "template file" is a bit confusing. Maybe calling web3api.gen.js the "Generation File" instead?